support-policy.md

```diff
@@ -0,0 +1,34 @@
+Support and Stability policy
+============================
+
+With regards to current and future releases the OpenSSL project has adopted the
+following policy:
+
+* Version 3.0 will be supported until 2023-09-07.
+* Version 1.1.1 will be supported until 2023-09-11 (LTS).
+* Version 1.0.2 is no longer supported. Extended support for 1.0.2 to gain
+access to security fixes for that version is
+[available](https://www.openssl.org/support/contracts.html).
+* Versions 1.1.0, 1.0.1, 1.0.0 and 0.9.8 are no longer supported.
```

Comment:
An idea discussed among OMC is that this table should be a separate web page that gets updated as new releases are made or dates change.  This policy itself would then be just the set of rules from which we derive the exact dates.